util/dvsim/SimCfg.py

```diff
@@ -122,6 +123,7 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         self.run_cmd = ""
 
         # Generated data structures
+        self.full_name = ""
```

Comment:
Could we set the `name` in `kmac_*_sim_cfg.hjson` to `kmac_masked` and remove `name` from `kmac_base_sim_cfg.hjson` (and likewise for sram)? I feel that may be an easier fix.

Reply:
we need a `name` without the variant.
The csr tests use `{name}_common_vseq`. We only have kmac_common_vseq. With your suggestion, we need to have `kmac_masked_common_vseq` and `kmac_unmask_common_vseq`. Likewise for other common tests.
Any thoughts?